lib/cc/analyzer/engine.rb

```diff
@@ -92,6 +92,10 @@ def docker_run_command
       end
 
       def env_file
+        if @config_json.size > 64 * 1024
```

Comment:
The limit is line length, I think it's `"ENGINE_CONFIG=#{@config_json}".size`, right?